src/components/styled-mnemonic-input/styled-mnemonic-input.tsx

```diff
@@ -0,0 +1,75 @@
+import { useClipboard } from '@react-native-clipboard/clipboard';
+import React, { FC, MutableRefObject, useCallback, useEffect, useRef, useState } from 'react';
+import { TextInput, TextInputProps, View } from 'react-native';
+
+import { emptyFn, mnemonicInputTimerToShow } from '../../config/general';
```

Comment:
`mnemonicInputTimerToShow` do not belong to general config.  
Jest declare it in this file